pkg/engine/apicall/apiCall.go

```diff
@@ -101,13 +101,16 @@ func (a *apiCall) executeK8sAPICall(ctx context.Context, path string, method kyv
 
 	jsonData, err := a.client.RawAbsPath(ctx, path, string(method), requestData)
 	if err != nil {
-		return nil, fmt.Errorf("failed to %v resource with raw url\n: %s: %v", method, path, err)
+		errMsg := fmt.Sprintf("failed to %v resource with raw url\n: %s: %v", method, path, err)
+		a.logger.Error(nil, errMsg, "name", a.entry.Name)
```

Comment:
strange to call `logger.Error` with `nil` why not pass `err` ?

Reply:
ok so should i create new pull request?

Reply:
why ? you can make changes in this one.